packages/esm-ward-app/src/config-schema.ts

```diff
@@ -1,5 +1,120 @@
-import { type ConfigSchema } from '@openmrs/esm-framework';
+import { type PersonAddress, Type, validator, validators, type ConfigSchema } from '@openmrs/esm-framework';
+import { type BentoElementType, bentoElementTypes } from './types';
 
-export const configSchema: ConfigSchema = {};
+const defaultWardPatientCard: WardPatientCardDefinition = {
+  card: {
+    id: 'default-card',
+    header: ['bed-number', 'patient-name', 'patient-age', 'patient-address'],
+  },
+  appliedTo: null,
+};
 
-export interface ConfigObject {}
+export const defaultBentoElementConfig: BentoElementConfig = {
```

Comment:
I don't understand why this is and why it has addressFields in it.

Reply:
It's supposed to be a common config object for various types of bentoElements. (See comment below)

Reply:
Yeah, this is a situation which the config schema doesn't handle very elegantly. Do we really have a use-case, though, for cards with different elements that display different parts of an address?